src/components/maps/BrowserMap.tsx

```diff
@@ -0,0 +1,32 @@
+import { MapProps } from './types';
+import { MapContainer, Marker, Popup, TileLayer } from 'react-leaflet';
+
+export default function Map({ height, markers, width }: MapProps): JSX.Element {
```

Comment:
The name of the component (`Map`) does not reflect the name of the module (`BrowserMap`) which is bad practice because it will cause any error messages to be confusing, amongst other reasons.

Reply:
Can you give example of where this is done correctly? i am not sure i understand what it means to reflect the name of the module

Reply:
Lets talk about this after my meeting!